config/webpack/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+
+const { join } = require('path');
+
+const shims = require('./webpack.shims');
+const basedir = join(__dirname, '../..');
+
+const config = require('../config');
+
+const prod = process.env.NODE_ENV === 'production';
+
+module.exports = merge(shims, {
+  mode: prod ? 'production' : 'development',
```

Comment:
nit: `prod` could be `isProduction` for clarity?

Reply:
Sure! Have a free :horse: too while I'm here :)